client/extensions/wp-job-manager/components/setup/page-setup/index.jsx

```diff
@@ -19,22 +21,88 @@ import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
 import ReduxFormToggle from 'components/redux-forms/redux-form-toggle';
 import SectionHeader from 'components/section-header';
+import { createPages } from '../../../state/setup/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isCreatingPages, shouldGoToNextStep } from '../../../state/setup/selectors';
 
 const form = 'extensions.wpJobManager.pageSetup';
 
+const validate = ( values, props ) => {
+	const { translate } = props;
```

Comment:
This could be moved into the arguments definition - `const validate = ( values, { translate } ) => {`